### DISCUSSION TITLE
GraphQL API - Archive ProjectV2Item

### DISCUSSION THREAD

**Discussion by SimonGRoze** [community/community#31555] [Apps, API and Webhooks] [answered]
Hi all! Is there a mutation to change the `isArchived` property on a `ProjectV2Item` using an API call?

Looking at the documentation for GraphQL mutations, there does not seem to be any way to directly modify a `ProjectV2Item` object. However, the Projects (classic) GraphQL API does support archiving `ProjectCard` items in a project using the `updateProjectCard` mutation, so I thought there might be a mutation in the ProjectsV2 API to perform an equivalent task.

If this is in the works, could anyone share a timeline for when this would be expected to be shipped?

If this does not exist, could you suggest an alternative method one can use to programmatically archive items on ProjectV2 boards? This would be useful e.g. to automatically clean up project boards using workflows.

Thank you all so much!

**Comment by jszwedko** [comment, marked as answer]
This would also be very useful for our workflows. I see it is possible to bulk archive from the UI but it seems to be hitting some private API, `https://github.com/memexes/<some id>/items/archive` with some `memexProjectItemIds` that I'm not sure where to find.

**Reply by jszwedko** [reply]
Actually, scratch that, a mutation does seem to exist now: https://docs.github.com/en/graphql/reference/mutations#archiveprojectv2item
